src/__type_tests__/chip-type-test.tsx

```diff
@@ -7,8 +7,8 @@ import Chip from '../chip';
 <Chip id="someid">hello</Chip>;
 <Chip onClose={() => {}}>hello</Chip>;
 
+// @ts-expect-error Chip only accepts string children
 <Chip>
-    {/* @ts-expect-error Chip only accepts string children */}
     <div>hello</div>
 </Chip>;
```

Comment:
this is curious, if I remove the `@ts-expect-error` the reported issue is:

> This JSX tag's `children` prop expects type `string` which requires multiple children, 
> but only a single child was provided.

It works as expected, but I find the error description misleading